libraries/AP_Camera/AP_Camera.h

```diff
@@ -26,6 +26,7 @@
 
 #define AP_CAMERA_FEEDBACK_DEFAULT_FEEDBACK_PIN -1  // default is to not use camera feedback pin
 
+
```

Comment:
Left-over?

Reply:
not related to what I did,  but I can demove that define and add it directly into the parameter default if you wish.

Reply:
I meant the extra line, but never-mind 🙂

Reply:
oh, sorry, done :)

Reply:
Well, now you need to squash it 😛

Reply:
Done :)